.github/workflows/builder.yml

```diff
@@ -0,0 +1,313 @@
+              "${docker_reg}/armhf-homeassistant:${tag_r}" \
+              "${docker_reg}/armv7-homeassistant:${tag_r}" \
+              "${docker_reg}/aarch64-homeassistant:${tag_r}"
+
+            docker manifest annotate "${docker_reg}/home-assistant:${tag_l}" \
+              "${docker_reg}/amd64-homeassistant:${tag_r}" \
+              --os linux --arch amd64
+
+            docker manifest annotate "${docker_reg}/home-assistant:${tag_l}" \
+              "${docker_reg}/i386-homeassistant:${tag_r}" \
+              --os linux --arch 386
+
+            docker manifest annotate "${docker_reg}/home-assistant:${tag_l}" \
+              "${docker_reg}/armhf-homeassistant:${tag_r}" \
+              --os linux --arch arm --variant=v6
+
+            docker manifest annotate "${docker_reg}/home-assistant:${tag_l}" \
+              "${docker_reg}/armv7-homeassistant:${tag_r}" \
+              --os linux --arch arm --variant=v7
+
+            docker manifest annotate "${docker_reg}/home-assistant:${tag_l}" \
+              "${docker_reg}/aarch64-homeassistant:${tag_r}" \
+              --os linux --arch arm64 --variant=v8
+
+            docker manifest push --purge "${docker_reg}/home-assistant:${tag_l}"
+          }
+
+          function validate_image() {
+            local image={1}
+            state="$(vcn authenticate --org home-assistant.io --output json docker://${image} | jq '.verification.status // 2')"
+            if [[ "${state}" != "0" ]]; then
+              echo "Invalid signature!"
+              exit 1
+            fi
+          }
+
+          for docker_reg in "homeassistant" "ghcr.io/home-assistant"; do
+            docker pull "${docker_reg}/amd64-homeassistant:${{ needs.init.outputs.version }}"
+            docker pull "${docker_reg}/i386-homeassistant:${{ needs.init.outputs.version }}"
+            docker pull "${docker_reg}/armhf-homeassistant:${{ needs.init.outputs.version }}"
+            docker pull "${docker_reg}/armv7-homeassistant:${{ needs.init.outputs.version }}"
+            docker pull "${docker_reg}/aarch64-homeassistant:${{ needs.init.outputs.version }}"
+
+            validate_image "${docker_reg}/amd64-homeassistant:${{ needs.init.outputs.version }}"
+            validate_image "${docker_reg}/i386-homeassistant:${{ needs.init.outputs.version }}"
+            validate_image "${docker_reg}/armhf-homeassistant:${{ needs.init.outputs.version }}"
+            validate_image "${docker_reg}/armv7-homeassistant:${{ needs.init.outputs.version }}"
+            validate_image "${docker_reg}/aarch64-homeassistant:${{ needs.init.outputs.version }}"
+
+            # Create version tag
+            create_manifest "${docker_reg}" "${{ needs.init.outputs.version }}" "${{ needs.init.outputs.version }}"
+
+            # Create general tags
+            if [[ "${{ needs.init.outputs.version }}" =~ d ]]; then
+              create_manifest "${docker_reg}" "dev" "${{ needs.init.outputs.version }}"
```

Comment:
Should dev and or latest always be updated?

Reply:
It's the same code as on azure and what people know to have 🤷‍♂️